lib/main.coffee

```diff
@@ -351,10 +351,10 @@ module.exports = Hydrogen =
             @wsKernelPicker = new WSKernelPicker (kernel) =>
                 @clearResultBubbles()
 
-                grammar = @editor.getGrammar()
+                grammar = kernel.grammar
```

Comment:
I'm defending against a race condition where the active grammar changes between the time that the `WSKernelPicker` is activated, and the time a kernel is selected.

Technically there should never happen. However the `WSKernelPicker` can have significant delays over poor network connections, and perhaps some tab-switching from another plugin can sneak into the event loop during that time.